kbase-extension/static/kbase/js/common/cellComponents/tabs/jobStatus/jobStatusTable.js

```diff
@@ -640,73 +677,92 @@ define([
                     this.handleJobInfo.bind(self)(jobInfo);
                 },
             });
+            this.jobManager.addEventHandler(jcm.MESSAGE_TYPE.RETRY, {
+                jobStatusTable_retry: (_, jobRetry) => {
+                    this.handleJobRetry.bind(self)(jobRetry);
+                },
+            });
         }
 
         // HANDLERS
 
         /**
          * Update the table with a new jobState object
-         * @param {object} jobState
+         * @param {array} jobArray - array of updated jobs
          */
-        updateJobStatusInTable(jobState) {
-            const rowIx = jobState.retry_parent || jobState.job_id;
+        updateTable(jobArray) {
+            jobArray.forEach((jobState) => {
```

Comment:
core functionality is the same, just takes an array of jobs instead of a single job